pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/index/startree/AggregationSpec.java

```diff
@@ -50,4 +52,10 @@ public boolean equals(Object o) {
   public int hashCode() {
     return _compressionType.hashCode();
   }
+
+  @Override
+  public String toString() {
+    return new ToStringBuilder(this, ToStringStyle.SHORT_PREFIX_STYLE).append("compressionType", _compressionType)
+        .toString();
```

Comment:
(minor) Is this for debugging purpose? We usually use the IDE auto-generated one
```suggestion
    return "AggregationSpec{" + "_compressionType=" + _compressionType + '}';
```

Reply:
I added this for more clarity in the minion logs. When indexes are constructed, the logs were printing the function column pair with an opaque object identity for the aggregation spec. It's a nice to have and can remove if this is going to interfere with development.

Reply:
I see. Just realize we are already using the `ToStringBuilder` in `StarTreeV2BuilderConfig`, so we should be good here